tidb-lightning: clarify an ambiguous description for `sorted-kv-dir` (#7962)
pingcap/docs: migrate-from-sql-files-to-tidb.md

# "tidb"：The "tidb" backend can also be used to import small volumes of data (less than 1 TiB). During the import, the target TiDB cluster can provide service normally. For the information about backend mode, refer to https://docs.pingcap.com/tidb/stable/tidb-lightning-backends.

backend = "local"

> # Sets the temporary storage directory for the sorted key-value files. The directory must be empty, and the storage space must be large enough to hold the dataset to be imported. For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage and exclusive I/O for the directory.

Author: ```suggestion
# Sets the temporary storage directory for the sorted key-value files. The directory must be empty, and the storage space must be greater than the size of the dataset to be imported. For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage and exclusive I/O for the directory.
```
